tests/conftest.py

```diff
@@ -461,7 +461,7 @@ def coverage_json():
         "totals": {
             "covered_lines": 6,
             "num_statements": 10,
-            "percent_covered": 60.0,
+            "percent_covered": 53.84615384615384615384615385,
```

Comment:
I'm more than a little concerned that _this_ number may shift a bit based on OS, CPU type, Python version, etc... we may want to finesse the fixture data so the figures come out slightly rounder.

Reply:
(Then again, since it's just `0.538461` repeating, maybe it won't be a problem.)

Reply:
You're probably right, IEEE 754 is a rabbit whole I'd rather not dig.